logger.js

```diff
@@ -108,7 +105,7 @@ module.exports = (externalLogger, logFolder, moduleName = "installer")=> {
       if (externalLogger) {externalLogger.setDisplaySettings(settings);}
     },
     external(evt, detail, table) {
-      appendToLog(evt, detail);
+      appendToLog(detail, evt);
```

Comment:
evt is not a user friendly message by itself, is it ? maybe changing it to something like `sending event: ${evt}`

Reply:
@santiagonoguez changes made, thanks